Dockerfile

```diff
@@ -65,23 +67,34 @@ RUN if ! getent group "$USER_GID"; then groupadd --gid "$USER_GID" dependabot ;
 
 ### RUBY
 
-# Install Ruby, update RubyGems, and install Bundler
-ENV BUNDLE_SILENCE_ROOT_WARNING=1
+ARG RUBY_VERSION=2.7.5
+ARG RUBY_INSTALL_VERSION=0.8.3
+
+ARG RUBYGEMS_SYSTEM_VERSION=3.2.20
 # Disable the outdated rubygems installation from being loaded
 ENV DEBIAN_DISABLE_RUBYGEMS_INTEGRATION=true
+
+ARG BUNDLER_V1_VERSION=1.17.3
+ARG BUNDLER_V2_VERSION=2.3.13
```

Comment:
I really like this update 🥇

💡 Seeing this change makes me wonder if we could take it further. What do you think about making these `ENV` variables, then pointing the references in our code ([example](https://github.com/dependabot/dependabot-core/blob/main/bundler/helpers/v1/build#L25-L27)) to `BUNDER_Vx_VERSION`? If you're interested, I think a followup PR would be a great fit for the change 🙂

Reply:
I don't fully recall the whole thing from my work on #4973, but I'm pretty sure if `BUNDLER_VERSION` is used is because RubyGems actually uses that ENV variable to activate the proper Bundler version. So if I understood what you mean, I think it would not work as expected. But if we introduced something like #4973 it wouldn't be needed anyways.